tools/travis/script.sh

```diff
@@ -16,7 +16,13 @@ tools/build_versions.py
 section_end "List.installed.dependencies"
 
 section "Test"
-pytest $TEST_ARGS skimage
+if [[ "$INSTALL_FROM_SDIST" == "1" ]]; then
+    pushd ..
+    pytest $TEST_ARGS --pyargs skimage
+    popd
+else
+    pytest $TEST_ARGS skimage
```

Comment:
Why do these two require different forms of `pytest`?

Reply:
`--pyargs` asks pytest to look in the installed module. This is necessary since the development directory is pretty bare and doesn't include any cythonized modules.